app/commands/v2/discard_draft.rb

```diff
@@ -62,28 +62,24 @@ def increment_live_lock_version
         lock_version.save!
       end
 
+      def document
+        @document ||= Queries::GetDocument.(content_id, locale)
```

Comment:
Can access these values straight from payload so hopefully we can kill of the need for individual methods for them?